r-package/grf/tests/testthat/test_boosted_regression_forest.R

```diff
@@ -0,0 +1,52 @@
+library(grf)
+
+set.seed(1)
+
+test_that("Boosted regression forest improves performance vs regular forest", {
```

Comment:
It would be nice to add one test around OOB prediction, since precomputing those predictions is a little trickier in the case of boosted forests.

Reply:
What sort of test would you suggest to test the OOB prediction?

Reply:
We could draw a test set from the same distribution as the training data, and check that the mean prediction error is not too different. We'll need to train a decent number of trees to make up for the fact that OOB prediction is only able to use the out-of-bag trees for each datapoint.

Reply:
Added this